resources/js/components/welcomevlb.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+    <div>
+        <p class="vlab-Text">Welcome to Vlab</p>
```

Comment:
Not sure we need this since a lading page exists. We route user to Dashboard or  referer page when the authenticate

Reply:
This is appearing on the login page